### ISSUE TITLE
determine and standardize a unit vs a individual unit. naming convention

### ISSUE THREAD

**Issue by SunnysideAaron** [SunnysideAaron/BKsDBTools#13]
how to determine we are talking about all spectras vs the single spectra of a single player.  what does com2us use?

**Comment by QuatZo** [comment]
Not sure if I understand correctly.
Com2uS uses 2 keys to determine monster:
- `unit_id`: current monster instance (it's being used for runes, artifacts, siege defenses, etc)
- `unit_master_id`: It says which base monster it is (i.e. 10101 is water fairy)

Basically:
- This Spectra: `unit_id`
- All Spectras: `unit_master_id`

**Comment by SunnysideAaron** [comment]
exactly. some of the sources have unit_id as com2us_id. ie different sources mix and match the id names.
